charts/kserve-resources/templates/configmap.yaml

```diff
@@ -511,6 +515,7 @@ data:
         "domainTemplate": "{{ .Values.kserve.controller.gateway.domainTemplate }}",
         "urlScheme": "{{ .Values.kserve.controller.gateway.urlScheme }}",
         "disableIstioVirtualHost": {{ .Values.kserve.controller.gateway.disableIstioVirtualHost }}
+        "disableIngressCreation": {{ .Values.kserve.controller.gateway.disableIngressCreation }},
```

Comment:
Can you also update the Kustomize manifests?

Reply:
Thanks. Updated